### PR TITLE
fixes #21

### LINKED ISSUE #6: NavbarRoutes
### Useful Resource  : **https://www.npmjs.com/package/react-router-dom**

### LINKED ISSUE #21: Create form page
Create form page that a user can post from.
use firebase to handle data after you finish the design.

Resource:
you can check out one of our previous bootcamp activities about firestore
[https://codesandbox.io/s/firestore-test-iipe8](https://codesandbox.io/s/firestore-test-iipe8)
Screen Shot 2021-03-23 at 5 37 15 PM

### PR DESCRIPTION
**Pull request by Ravyar19**

### CODE REVIEW

**Review by murtaja1 — APPROVED** [review]

File: src/components/Form/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import useStyles from './Style'
+import TextField from '@material-ui/core/TextField';
+
+
+
+const Form = () => {
+    const classes = useStyles()
+
+
+    return (
```
Comment (Othmanosx) [inline src/components/Form/Form.jsx:12]: Add NavBar component

File: src/components/Form/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                    <p className={classes.desc}>Please enter the email to display on your submission</p>
+                    <TextField id="outlined-basic" label="Email" variant="outlined" size="large" className={classes.input}/>
+                    <br/>
+                    <Button variant="contained" size="large" className={classes.button}  color="primary">Submit</Button>
+
+                </Grid>    
+            </Grid>
+           
+              
+       </form>
```
Comment (Othmanosx) [inline src/components/Form/Form.jsx:69]: Add footer component

File: src/components/Form/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    <h4 className={classes.title} style={{
+                        marginTop:"8.5rem"
+                    }}>Last Name</h4>
+                    <p className={classes.desc}>Please enter your last name(This will appear on your submission)</p>
+                    <TextField id="outlined-basic" label="Last Name" variant="outlined" size="large" className={classes.input}/>
+                    <h4 className={classes.title}>Email</h4>
+                    <p className={classes.desc}>Please enter the email to display on your submission</p>
+                    <TextField id="outlined-basic" label="Email" variant="outlined" size="large" className={classes.input}/>
+                    <br/>
+                    <Button variant="contained" size="large" className={classes.button}  color="primary">Submit</Button>
```
Comment (Othmanosx) [inline src/components/Form/Form.jsx]: center the submit button

**Review by Othmanosx — COMMENTED** [review]
Looks OK, you can work on improving the responsive design more.

**Review by maqalaqil — APPROVED** [review]
lgtm

File: src/components/Form/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+} from "@material-ui/pickers"
+
+const Form = () => {
+  const [selectedDate, setSelectedDate] = React.useState(
+    new Date("2014-08-18T21:11:54")
+  )
+
```
Comment (murtaja1) [inline src/components/Form/Form.jsx:18]: have all your state in one useState.

**Review by murtaja1 — APPROVED** [review]

**Pull request merged**